Prototope/Layer.swift

```diff
@@ -459,7 +516,20 @@ public class Layer: Equatable, Hashable {
 			return emitter !== particleEmitter
 		}
 	}
+	
+	
+	// TODO(jb): Port touches / gestures to OS X? What makes sense here?
```

Comment:
Yeah, I’ve played around with those a bit before, but wasn’t sure if that’d really make sense for prototyping OS X apps. I’ll probably give it a shot eventually though!